activestate.yaml

```diff
@@ -17,6 +17,10 @@ constants:
       GOPATH="${GOPATH:=`go env GOPATH`}"
       GOROOT="${GOROOT:=`go env GOROOT`}"
       PATH="${GOROOT}/bin:${GOPATH}/bin:${PATH}"
+      if [ "$CI" == "true" ]
+      then
+        GO_BUILD_TAGS="CI"
+      fi
```

Comment:
How is this related to this PR?

Reply:
Indirectly based on the comment here: https://github.com/ActiveState/cli/blob/master/cmd/state/errors.go#L101. Since this error was reported from a state tool version that was built on CI.